objects/rct2ww/ride/rct2.ww.mandarin.json

```
@@ -58,6 +58,7 @@
             "nl-NL": "Mandarijneendbootjes",
             "sv-SE": "Svansjön",
             "zh-CN": "鸳鸯水上游乐设施",
+            "ja-JP": "鴛鴦ボート",
```

Uncommon kanji 「鴛鴦」; use 「オシドリ」 instead.
```suggestion
            "ja-JP": "オシドリのボート",
```